pkg/istio-agent/xds_proxy.go

```diff
@@ -62,6 +63,8 @@ import (
 	"istio.io/istio/security/pkg/nodeagent/caclient"
 	"istio.io/istio/security/pkg/pki/util"
 	"istio.io/pkg/log"
+
+	"go.opencensus.io/zpages"
```

Comment:
any idea how expensive this is? Safe to always enable?

Reply:
Just some extra handlers - using the existing data.